### PR TITLE
use "main" instead of "master" in simple synth doc

### PR DESCRIPTION
**Pull request by hirehamir**
Checklist — To help your pull request get merged faster, please do the following:

1. - [x] Provide a summary of your changes — say what problem you are fixing, what files are changed, and what you've done. This doesn't need to be hugely detailed, as we can see exact changes in the "Files changed" tab.
This pull request changes an article so that it uses "main" instead of "master"
1. - [x] Review the results of the automated checking we run on every PR and fix any problems reported (see the list of checks near the bottom of the PR page). If you need help, please ask in a comment!
1. - [x] Link to any other resources that you think might be useful in reviewing your PR.

The page this pull request updates if we merge it is https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API/Simple_synth.

### CODE REVIEW

**Comment by CreaTorAlexander** [comment]
Why did you changed it from master to main? Is there an issue related to it, if yes can you link this related issue.

File: files/en-us/web/api/web_audio_api/simple_synth/index.html [hunk 1]
```diff
@@ -551,16 +551,16 @@ <h4 id="Stopping_a_tone">Stopping a tone</h4>
...
+<p>The volume slider in the settings bar provides a simple interface to change the gain value on the main gain node, thereby changing the loudness of all playing notes. The <code>changeVolume()</code> method is the handler for the {{event("change")}} event on the slider.</p>
 
 <pre class="brush: js">function changeVolume(event) {
-  masterGainNode.gain.value = volumeControl.value
+  mainGainNode.gain.value = volumeControl.value
 }
 </pre>
 
-<p>This sets the value of the master gain node's <code>gain</code> {{domxref("AudioParam")}} to the slider's new value.</p>
+<p>This sets the value of the main` gain node's <code>gain</code> {{domxref("AudioParam")}} to the slider's new value.</p>
```
Comment (chrisdavidmills) [inline files/en-us/web/api/web_audio_api/simple_synth/index.html:563]: they is a stray backtick here that needs removing.

**Comment by chrisdavidmills** [comment]
@CreaTorAlexander "master" is culturally-insensitive language, and ideally we should be removing it anywhere it is found.

I'm going to give this to @Rumyra to review, as this is her domain. There might be an example repo that needs updating along with this page too...

File: files/en-us/web/api/web_audio_api/simple_synth/index.html (hunk 1 above)
Comment (hamirmahal) [reply files/en-us/web/api/web_audio_api/simple_synth/index.html]: Ah, I missed that. I just fixed it with 5cbe03b. Thanks for catching that.

**Review by Rumyra — APPROVED** [review]

**Comment by Rumyra** [comment]
This is great thank you so much @hamirmahal 👍

@chrisdavidmills there's no example repo code for this - but instead of blocking this PR I'll just make a note. I already have a task of going over the web audio demos 🎶

**Comment by chrisdavidmills** [comment]
cool, cheers @Rumyra !

**Comment by hamirmahal** [comment]
You're welcome, @Rumyra! Happy to help.

**Pull request merged**